openqa-investigate

```diff
@@ -26,6 +26,7 @@ runjq() {
     rc=$?
     set -e
     (( "$rc" == 0 )) && echo "$output" && return
+    output=$(echo "$output" | head -15)
```

Comment:
please replace this magic number. How about

```suggestion
jq_output_limit="${jq_output_limit:-15}"
…
    output=$(echo "$output" | head -$jq_output_limit)
```

Reply:
done